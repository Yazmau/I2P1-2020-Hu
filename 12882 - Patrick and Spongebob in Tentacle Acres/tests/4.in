1 48
63 34 52 79 94 32 77 9 64 41 33 46 95 36 14 6 34 17 80 72 73 47 11 99 92 87 40 57 88 23 5 43 6 64 16 90 45 69 60 77 60 53 44 41 42 13 90 35
37
1 1 2 2
1 1 6 36
1 1 6 19
1 1 43 44
1 1 3 45
1 1 31 32
1 1 5 48
1 1 30 48
1 1 25 29
1 1 23 41
1 1 29 48
1 1 5 41
1 1 29 37
1 1 6 17
1 1 38 40
1 1 23 31
1 1 22 42
1 1 4 26
1 1 24 24
1 1 12 32
1 1 12 28
1 1 26 32
1 1 12 31
1 1 44 47
1 1 7 39
1 1 47 48
1 1 41 45
1 1 48 48
1 1 16 48
1 1 27 41
1 1 8 29
1 1 31 32
1 1 30 33
1 1 7 41
1 1 16 35
1 1 1 48
1 1 41 41
