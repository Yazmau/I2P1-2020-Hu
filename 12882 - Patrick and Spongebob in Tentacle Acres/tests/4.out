34
99
95
44
99
43
99
90
92
99
90
99
90
95
77
99
99
99
99
99
99
88
99
90
99
90
60
35
99
90
99
43
43
99
99
99
60
